src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,63 @@
 package core.basesyntax;
 
+import java.util.LinkedList;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private LinkedList<Entry<K, V>>[] buckets;
+    private int size;
+
+    private static class Entry<K, V> {
```

Comment:
The `Entry` class should have access modifiers for its fields and constructor to follow the encapsulation principle.